### DISCUSSION TITLE
create a rpm release file for OpenSUSE

### DISCUSSION THREAD

**Discussion by Riyadulmollick** [OpenBangla/OpenBangla-Keyboard#223] [Ideas]
OpenBangla-Keyboard already have a rpm release file for the latest Fedora 34.

For openSUSE supports, we just need to change the `Requires:`  section on **rpmbuild>spec**  file.
Then just normally use rpmbuild to create the rpm file for openSUSE.

It is the same as Fedora/CentOS, but the required files are changed.

**Comment by Dyrimon** [comment]
👍🏼 bump for this.

The current bash script shows error: "Sorry, this script was not able to identify your OS distribution." in Tumbleweed.
It should be noted that openSUSE favors KDE Plasma, so fcitx support should be merged to the master branch (as the [current KDE configuration](https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Configure-on-KDE) doesn't work seamlessly with Plasma 5.x).
